src/main/java/net/javadiscord/javabot/data/config/guild/HelpConfig.java

```diff
@@ -93,6 +99,12 @@ public class HelpConfig extends GuildConfigItem {
 	 */
 	private int minimumMessageLength = 10;
 
+	/**
+	 * The message-embed's footnote of an unformatted-code-replacement.
+	 * Issued by {@link net.javadiscord.javabot.systems.help.AutoCodeFormatter}
+	 */
+	private String autoFormatInfoMessage = "This message has been formatted automatically.";
```

Comment:
It might be a good idea to mention that this can be disabled using `/preferences`.